CHANGELOG.md

```diff
@@ -31,6 +31,7 @@
 - Upgrade the Helm dependency version from `v3.0.1` to `v3.0.2`. ([#2621](https://github.com/operator-framework/operator-sdk/pull/2621)) 
 - Changed the scaffolded `serveCRMetrics` to use the namespaces informed in the environment variable `WATCH_NAMESPACE` in the MultiNamespace scenario. ([#2603](https://github.com/operator-framework/operator-sdk/pull/2603)) 
 - Improve skip metrics logs when running the operator locally in order to make clear the information for Helm based operators. ([#2603](https://github.com/operator-framework/operator-sdk/pull/2603))
+- Bumped operator-registry dependency version to `v1.6.0` to update `bundle create` behavior. Includes adding new optional `output-dir` flag to specify a result bundle directory.
```

Comment:
Could you please update it as follows?

In the changes section: 

```
- Upgrade the `operator-registry` dependency version from ` v1.5.7`to `v1.6.0` to update `bundle create` behaviour. ([#2662](https://github.com/operator-framework/operator-sdk/pull/2662)) 
```
In the add section: 

```
- Add the new flag `output-dir` to the command `operator-sdk bundle create`. ([#2662](https://github.com/operator-framework/operator-sdk/pull/2662)) 
```